### DISCUSSION TITLE
BISON "Vectors are not the same length" error

### DISCUSSION THREAD

**Discussion by kloibz** [idaholab/moose#19165] [Q&A Getting Started] [answered]
<img width="1775" alt="Screen Shot 2021-10-20 at 5 31 57 PM" src="https://user-images.githubusercontent.com/88003887/138190615-25d46086-843a-4fa5-a35f-16bf6cf230da.png">
New BISONer, first time poster:

I am trying to run a metallic fuel simulation, but I am getting the above error message saying that "Vectors are not the same length" and I am unsure how to debug the error. I am recreating a file based off of a template file that has run and I know that my build of BISON is good because I am able to run the assessment files with no issue. So far I have gone line by line through my input file comparing and trying to find differences, but I am still stuck. Any debugging suggestions or useful interpretations from the error code that could point me to problem area?

Thanks!

**Comment by jiangwen84** [comment, marked as answer]
@kloibz Since this is a Bison related question, I suggest you send an email directly to bison-users@inl.gov We will answer your question there. Thanks.

**Reply by kloibz** [reply]
Okay, thanks!
